experiments/tf_trainer/common/tfrecord_simple.py

```diff
@@ -60,17 +79,15 @@ def _read_tf_example(
     The input TF Example has a text feature as a singleton list with the full
     comment as the single element.
     """
-    DEFAULT_VALUES = {tf.string: '', tf.float32: -1.0, tf.int32: -1}
 
     keys_to_features = {}
     keys_to_features[self._text_feature] = tf.FixedLenFeature([], tf.string)
-    for label, dtype in self._labels.items():
-      keys_to_features[label] = tf.FixedLenFeature([], dtype,
-                                                   DEFAULT_VALUES[dtype])
+    for label in self._labels:
+      keys_to_features[label] = tf.FixedLenFeature([], tf.float32, -1.0)
     parsed = tf.parse_single_example(
         record, keys_to_features)  # type: Dict[str, types.Tensor]
 
-    features = {self._text_feature: parsed[self._text_feature]}
+    features = {'text': parsed[self._text_feature]}
```

Comment:
It is probably better to make it a class static var (so we can change it in one place should we need to), avoiding lots of string literal dependencies, and perhaps it should be noted in the class doc-string that the output text feature will always be called 'text' no matter the value of `self._text_feature` ?

Reply:
Sending a separate email to explain the use of 'text'

Reply:
Thanks, that helps. My summary, for github, is that is that consumers of the model may have different ways they provide the model with the text feature at evaluation time. And you wanted to make it fixed to 'text' for one of them. I think it might be better to do that as a data-set pre-processing step, or provide it as a command line argument (I guess I'd name the command line argument something like "renamed_text_field")? I'm a little skeptical about this being the right place for this abstraction, but it may be convenient to be able to change the output model like this, and it's not terribly painful.

Reply:
This has now been abstracted to a variable in base_model.